org.gnome.Hamster.GUI.desktop

```diff
@@ -0,0 +1,20 @@
+[Desktop Entry]
```

Comment:
desktop file is upstream, why adding it here?

Reply:
removed.